sklearn/preprocessing/_encoders.py

```diff
@@ -96,8 +126,14 @@ def _fit(self, X, handle_unknown='error'):
                         msg = ("Found unknown categories {0} in column {1}"
                                " during fit".format(diff, i))
                         raise ValueError(msg)
+                if return_counts:
+                    category_counts.append(_get_counts(Xi, cats))
+
             self.categories_.append(cats)
 
+        return {'category_counts': category_counts,
```

Comment:
So this will always be returned but if ``return_count=False`` then the value will be ``None``?
I'm not sure what's weirder, changing the return value of a method with a keyword argument, or not changing the return value if the keyword argument is called ``return_...`` ;)

Reply:
I think only returning when `return_count=True` is nicer.